kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/test/java/org/kie/server/integrationtests/common/KieServerContainerCRUDIntegrationTest.java

```diff
@@ -150,22 +150,6 @@ public void testDisposeNonExistingContainer() throws Exception {
     }
 
     @Test
-    public void testUpdateReleaseIdForNotExistingContainer() throws Exception {
```

Comment:
@karreiro Please try to update the tests instead of deleting them. In this scenario Kie server should return error in case we try to update not existing container (according to new behaviour).

Reply:
Good catch, @sutaakar. Fixed.